courses-app/src/components/Courses/components/SearchBar/SearchBar.jsx

```diff
@@ -1,20 +1,30 @@
 import "./SearchBar.css";
 import Button from "../../../../common/Button/Button";
-import Courses from "../../Courses";
+// import Courses from "../../Courses";
 
 const SearchBar = ({
   setCoursesList,
   coursesList,
   setIsCreateOpened,
   isCreateOpened,
+  setFilteredList,
 }) => {
   const addCourse = (course) => {
     setCoursesList([...coursesList, course]);
   };
   const handleSearch = () => {
     const value = document.getElementById("searchInput").value;
-    console.log(value);
-    console.log(coursesList);
+
+    const filteredList = coursesList.filter((element) => {
+      return (
+        element.name === value.toString() || element.id === value.toString()
```

Comment:
Find a better solution for filtering. So the user will be able to find the course with the name "JavaScript" by just entering "java" or "script" etc.